modules/apps/portal-vulcan/portal-vulcan-api/src/test/java/com/liferay/portal/vulcan/util/LocalizedMapUtilTest.java

```diff
@@ -117,6 +118,66 @@ public void testMergeLocalizedMap() {
 		Assert.assertEquals("hello", map.get(LocaleUtil.US));
 	}
 
+	@Test
+	public void testPopulateI18nMap() {
+
+		// Do not populate i18nMap if both defaultLanguageId and
+		// siteDefaultValue are undefined
```

Comment:
// Do not populate i18nMap if both default Language ID and site default value are undefined

Reply:
You wrote them in separate words in the other comment

Reply:
but since the ci passed, we can forward and see what brian's think about the comments.